skiko/src/commonMain/kotlin/org/jetbrains/skia/Matrix22.kt

```diff
@@ -0,0 +1,47 @@
+package org.jetbrains.skia
+
+import org.jetbrains.skia.impl.InteropPointer
+import org.jetbrains.skia.impl.InteropScope
+import org.jetbrains.skia.impl.withResult
+
+/**
+ * 2x2 matrix.
+ */
+class Matrix22(vararg mat: Float) {
+    /**
+     * Matrix elements are in row-major order.
+     */
+    internal val mat: FloatArray
```

Comment:
It would be better to make this public. Because otherwise without matrix operations it will be useless. Also, it will be good, If you make `Matrix44.mat` public too.

Reply:
> Because otherwise without matrix operations it will be useless

Ah, sorry. It would be useless, when we get matrix (for example, canvas.matrix). I encountered on this issues in the past. It isn't useless, when we set matrix. But it is better to fix it anyway

Reply:
Done for 2x2 and 4x4, so that all three expose this as a public field